src/dmd/dinterpret.d

```diff
@@ -4963,6 +4963,12 @@ public:
             fd = ve.var.isFuncDeclaration();
             assert(fd);
 
+            // If `HookTraceImpl` is found, resolve the underlying hook and replace `e` and `fd` with it.
+            removeHookTraceImpl(e, fd);
+
+            // The following code is used to intercept lowered hooks and to run their corresponding
+            // CTFE action instead. This is needs to be done as the ctfe is ran after all the lowering and optimizations
+            // 1
```

Comment:
I think this comment is in the wrong spot.

Reply:
I placed this comment here because in the future PR more hooks will be added here. but maybe it is not needed?

(also I will remove the line with the `1`)

Reply:
Probably just better to remove it, and re-add it if needed in the future.